src/python/grpcio_health_checking/setup.py

```diff
@@ -40,17 +53,28 @@
     '': '.',
 }
 
-SETUP_REQUIRES = (
-    'grpcio-tools>={version}'.format(version=grpc_version.VERSION),)
-
 INSTALL_REQUIRES = ('protobuf>=3.3.0',
                     'grpcio>={version}'.format(version=grpc_version.VERSION),)
 
-COMMAND_CLASS = {
-    # Run preprocess from the repository *before* doing any packaging!
-    'preprocess': health_commands.CopyProtoModules,
-    'build_package_protos': health_commands.BuildPackageProtos,
-}
+try:
+    # ensure we can load the _pb2_grpc module:
+    from grpc_health.v1 import health_pb2_grpc as _pb2_grpc
+    # if we can find the _pb2_grpc module, the package has already been built.
+    SETUP_REQUIRES = ()
+    COMMAND_CLASS = {
+        # wire up commands to no-op not to break the external dependencies
```

Comment:
Who's "the external dependencies"? Do we have to do this? Is this something that can be cleaned up in the future?

Reply:
https://github.com/grpc/grpc/blob/e52772451a7bbf4f3f7b72cfc369781fd74a6930/tools/run_tests/artifacts/build_artifact_python.sh#L69

Reply:
The more I think about it the more I think we should just ditch the current build process right in the file I linked above :)

Reply:
Take a stab at it! The worst that happens is that we learn that it's a larger work item than anticipated and fall back on this instead. :-)